### ISSUE TITLE
Select github repositories from dropdown

### ISSUE THREAD

**Issue by Akirtovskis** [ledokku/ledokku#295]
# Feature request

Once https://github.com/ledokku/ledokku/pull/265 is merged, would be nice to have ability for user to select their public repos from gh

## Is your feature request related to a problem? Please describe.

A clear and concise description of what you want and what your use case is.
